README.md

```diff
@@ -97,6 +97,8 @@ There’s also a <a href="https://vimeo.com/168648012">fantastic video</a> on ho
 
 ## Quick start
 
+**We strongly recommend that you use yarn over npm because yarn installs dependencies in a stable and predictable way. As a bonus, you'll get faster installation and cleaner console!**
```

Comment:
I don't think we need to make such a highlighted statement, imo the section below should be good enough. There's more important stuff for beginners to worry about rather than "Wtf is yarn"?